yapf/yapflib/style.py

```diff
@@ -54,6 +54,13 @@ def SetGlobalStyle(style):
 _STYLE_HELP = dict(
     ALIGN_CLOSING_BRACKET_WITH_VISUAL_INDENT=textwrap.dedent("""\
       Align closing bracket with visual indentation."""),
+    ALIGN_ASSIGNMENT=textwrap.dedent("""\
```

Comment:
Please add the examples to these knobs.

Also the knobs should be alphabetized.